devtools/maven/src/main/java/io/quarkus/maven/ListExtensionsMojo.java

```diff
@@ -20,7 +19,8 @@ public void execute() {
         MojoUtils.loadExtensions().stream()
                 .sorted((o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName()))
                 .forEach(ext -> getLog()
-                        .info("\t * " + ext.getName() + " (" + ext.getGroupId() + ":" + ext.getArtifactId() + ")"));
+                        .info("\t * " + ext.getName() + " (" + ext.getSimplifiedArtifactId() + ")"
```

Comment:
Personally, i prefer String.format() here rather than concatenation as it makes it easier to read both the string content and what's going in to each position.

Reply:
this class remove installed extensions. I propose to add option to listExtension. by default it list installable extensions but i add a `-a` or `--all` to display all extensions including the installed one